beacon_node/store/src/hot_cold_store.rs

```diff
@@ -129,6 +137,9 @@ impl<E: EthSpec> BlockCache<E> {
             .get(block_root)
             .and_then(|map| map.get(column_index).cloned())
     }
+    pub fn get_data_column_custody_info(&mut self) -> Option<DataColumnCustodyInfo> {
```

Comment:
This doesn't require a mutable reference to `self` i think?

```suggestion
    pub fn get_data_column_custody_info(&self) -> Option<DataColumnCustodyInfo> {
```